test/specs/simulation.vehicles.spec.js

```diff
@@ -0,0 +1,22 @@
+const { generateRandomVehicles } = require('../../server/simulation/vehicles');
+const { randomDavAddress, randomDroneModel, randomCoords, randomRating, randomMissionsCompleted } = require('../../server/simulation/random');
+
+describe('randomVehicle()', () => {
+  test('returns an array of size 4', () => {
```

Comment:
The idea is to divide this to as many tests as possible, each testing one behavior of the function (all of these tests are grouped into one group - the `describe`).

You will then want to test on all the assertions mentioned in #36.

For example, here are the 2nd and 3rd assertions from #36:
```javascript
describe('randomVehicle()', () => {

  test('returns an array containing as many objects as the integer passed as the first argument (vehicleCount)', () => {
    const count = 5;
    const myVehicle = generateRandomVehicles(count, randomCoords, 2000);
    expect(myVehicle).toBeInstanceOf(Array); //check that it's an array
    expect(myVehicle).toHaveLength(count); // check that the length is right
  });

  test('returns an array containing 4 objects if undefined is passed as the first argument (vehicleCount)', () => {
    const myVehicle = generateRandomVehicles(undefined, randomCoords, 2000);
    expect(myVehicle).toBeInstanceOf(Array); //check that it's an array
    expect(myVehicle).toHaveLength(4); // check that the length is right
  });

});
```

Yes, it is a bit more verbose to write, but now your tests read like documentation:

<img width="690" alt="2__tal_tals-macbook-pro-2____dropbox_mysite_annyang__zsh_" src="https://user-images.githubusercontent.com/926653/30023340-92a3c1e8-9177-11e7-91ff-5a9f7da4b351.png">

More importantly, if something breaks, you immediately know what broke and not just a vague idea of which function broke. This is very useful when many different contributors are working on code they are not familiar with.